test/test_manifest_providers.py

```diff
@@ -46,9 +49,46 @@ def test_git_source():
     assert '' == package_path
     assert '<version>0.5.7</version>' in package_xml
 
-
+# mock_get_url_contents is used to mock out the '_get_url_contents' method in
+# the rosdistro.manifest_provider.github module.  Instead of going out to github
+# to get data (which can get rate-limited in Travis), it instead pulls a canned
+# example from some local files.
+def mock_get_url_contents(req):
+    import re
+    import sys
+
+    # For python3, look for the 'str' type; for Python 2, the 'unicode' type
+    if sys.version_info[0] == 3:
+        text_type = str
+    else:
+        text_type = unicode
```

Comment:
Please use a check which doesn't "break" in the future e.g. with Python 4, e.g.:

```
try:
    text_type = unicode
except NameError:
    text_type = str
```

Reply:
Sure, makes sense.  #110